pip_test.go

```diff
@@ -0,0 +1,380 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-cli-go/artifactory/commands/pip"
+	piputils "github.com/jfrog/jfrog-cli-go/artifactory/utils/pip"
+	"github.com/jfrog/jfrog-cli-go/inttestutils"
+	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
+	"github.com/jfrog/jfrog-cli-go/utils/tests"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+)
+
+type PipCmd struct {
+	Command string
+	Options []string
+}
+
+func TestPipInstall(t *testing.T) {
+	// Init pip.
+	initPipTest(t)
```

Comment:
It looks like the *initPipTest* function is not invoked from where the other init functions are invoked.